src/components/charts/ChartsHelper.js

```diff
@@ -0,0 +1,61 @@
+
+const percentageFormatter = showPercentage => v => (showPercentage ? `${v}%` : v);
+
+const addPercentage = v => (percentageFormatter(true)(v));
+
+const calculateBarChartData = (data, showPercentage, percentageFixedPoint) => {
+  if (showPercentage) {
+    const sum = data.reduce((a, entry) => a + entry.value, 0);
+    let percentRemaining = 100;
+    return data.map((entry, index, array) => {
+      let percentage;
+      if (index < array.length - 1) {
+        percentage = (entry.value * 100) / sum;
+      } else {
+        percentage = percentRemaining;
+      }
+      percentage = Number(Number(percentage).toFixed(percentageFixedPoint));
+      percentRemaining -= percentage;
+      return Object.assign({ percentage }, entry);
+    });
+  }
+  return data;
+};
+
+const getPercentageData = (chartData, percentageFixedPoint) => {
```

Comment:
This function looks pretty much the same as the function above it, can we combine them somehow?